test/helpers/kubectl.go

```diff
@@ -184,7 +190,7 @@ func init() {
 			defaultHelmOptions["global.tag"] = version
 			// This always works because SplitContainerURL would not return
 			// isFullyQualified == true otherwise
-			parts := strings.SplitN(image, "/", 1)
+			parts := strings.SplitN(image, "/", 2)
```

Comment:
i'm not sure i get this change, what exactly is going on here?

Reply:
ok yeah sorry i should have added more description in the commit message. basically this chunk of code is for extracting docker registry (e.g. `quay.io/cilium`) from CILIUM_IMAGE env variable. say CILIUM_IMAGE is set to `quay.io/cilium/cilium:latest`. then:

- `registry, image, version, isFullyQualified := SplitContainerURL(v)` splits it into `quay.io`, `cilium/cilium`, and `latest`.
- get the "organization" part from `cilium/cilium` by splitting it by slashes.
- set `global.registry` to `quay.io/cilium`.

the issue is that `SplitN`returns the string itself if the third argument is set to one.